supereeg/model.py

```diff
@@ -490,10 +493,12 @@ def save(self, fname, compression='blosc'):
     def get_slice(self, inds, inplace=False):
         """
         Indexes model object data
+
         Parameters
         ----------
         inds : scalar, list, or numpy array
-            locations you wish to index (relative to model.get_locs())
+            Locations you wish to index (relative to model.get_locs())
```

Comment:
Is this correct?  I thought we were indexing directly on model.locs (at least that's what we do for Brain objects, right?) We should just make sure the documentation is correct...and ideally Models and Brains should be indexed the same way.

Note: I'm pretty sure that model.locs and model.get_locs() return the same thing-- if so, we can just make the documentation for `Model.get_slice` match the documentation for `Brain.get_slice` without changing the code here.

Reply:
yep, mo.locs and mo.get_locs() is the same, so I changed the documentaiton